docs/config_samples/dataset_types/ga_s2_ard.yaml

```diff
@@ -13,18 +13,19 @@ measurements:
 - name: 'pixel_quality'
   aliases: [mask, CFmask]
```

Comment:
CFmask should be Fmask.  The C version of Fmask was not used in the product creation.